tutorials/introductory/customizing.py

```diff
@@ -35,20 +35,10 @@
 # -----------------------
 #
 # You can create custom styles and use them by calling `.style.use` with
-# the path or URL to the style sheet. Additionally, if you add your
-# ``<style-name>.mplstyle`` file to ``mpl_configdir/stylelib``, you can reuse
-# your custom style sheet with a call to ``style.use(<style-name>)``. By
-# default ``mpl_configdir`` should be ``~/.config/matplotlib``, but you can
-# check where yours is with `matplotlib.get_configdir()`; you may need to
-# create this directory. You also can change the directory where Matplotlib
-# looks for the stylelib/ folder by setting the :envvar:`MPLCONFIGDIR`
-# environment variable, see :ref:`locating-matplotlib-config-dir`.
-#
-# Note that a custom style sheet in ``mpl_configdir/stylelib`` will override a
-# style sheet defined by Matplotlib if the styles have the same name.
+# the path or URL to the style sheet.
 #
 # For example, you might want to create
-# ``mpl_configdir/stylelib/presentation.mplstyle`` with the following::
+# ``../images/presentation.mplstyle`` with the following::
```

Comment:
The `../` may be a bit confusing. Going up a directory from where? What's up there?
Using the assumption of a simple script, one usually doesn't have subfolders. In such a case the mplstyle file would be next to the script. Since that doesn't make it clear that a path is used, I propose to go with

```suggestion
# ``./images/presentation.mplstyle`` with the following::
```